book/src/integrate/android_tasks.md

```diff
@@ -25,10 +25,18 @@ Next, add these lines near the bottom of `android/app/build.gradle`:
     tasks.register("cargoBuild$taskPostfix", Exec) {
         // Until https://github.com/bbqsrc/cargo-ndk/pull/13 is merged,
         // this workaround is necessary.
-        commandLine 'sh', '-c', """cd ../../$crate && \
-        ANDROID_NDK_HOME="$ANDROID_NDK" cargo ndk \
+        
+        workingDir "../../$crate"
+        environment "ANDROID_NDK_HOME", "$ANDROID_NDK"
+        if (org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.currentOperatingSystem.isWindows()) {
+            commandLine 'cmd', '/C', """cargo ndk \
             -t armeabi-v7a -t arm64-v8a \
             -o ../android/app/src/main/jniLibs build $profileMode"""
+        } else {
+            commandLine 'sh', '-c', """cargo ndk \
+            -t armeabi-v7a -t arm64-v8a \
+            -o ../android/app/src/main/jniLibs build $profileMode"""
```

Comment:
Shall we extract common strings, i.e. the "cargo ndk ..." strings, into a variable? Because across the two commands, only cmd/sh /C/-c differs.

Reply:
> Shall we extract common strings, i.e. the "cargo ndk ..." strings, into a variable? Because across the two commands, only cmd/sh /C/-c differs.

sure